WebApplication/Backend/Util/DateSubstitute.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebApplication.Backend.Util
+{
+    public class DateSubstitute
+    {
+        int year;
+        int month;
+        int day;
+
+        public DateSubstitute(string year, string month, string day)
+        {
+            this.year = int.Parse(year);
+            this.month = int.Parse(month);
+            this.day = int.Parse(day);
+        }
+
+        public DateSubstitute()
+        {
+
+        }
+
+        public int Day { get => day; set => day = value; }
+        public int Month { get => month; set => month = value; }
+        public int Year { get => year; set => year = value; }
+
+        public List<DateSubstitute> SortDateSubstitutes(List<DateSubstitute> datesToSort)
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.icomparable?view=net-5.0
https://www.c-sharpcorner.com/UploadFile/afenster/lambda-expressions-are-wonderful/
This could do it in like two lines and regular DateTime class has already implemented IComparable

Reply:
This does the job. I don't have time to do the functionality again, some other time. :)

Reply:
Then this will be merged into develop some other time

Reply:
Some other time has come :D